node-operators/network-node/index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Network Node
+description: Learn how to set up and run a Tanssi-powered network using Docker or Systemd, which allows you to have your own RPC endpoint to interact with your appchain.
```

Comment:
```suggestion
description: Learn how to set up and run a node on a Tanssi-powered network using Docker or Systemd, which allows you to host your own RPC endpoint for chain interaction.
```

We have to avoid using appchain now